tests/unit/test-util/test.file-filter.js

```diff
@@ -74,12 +74,22 @@ describe('util/file-filter', () => {
 
     it('ignore artifactsDir and its content', () => {
       const filter = new FileFilter({
+        sourceDir: '.',
         artifactsDir: 'artifacts',
       });
       assert.equal(filter.wantFile('artifacts'), false);
       assert.equal(filter.wantFile('artifacts/some.js'), false);
     });
 
+    it('does not ignore an artifactsDir outside of sourceDir', () => {
+      const filter = new FileFilter({
+        artifactsDir: '.',
+        sourceDir: 'dist',
+      });
+      assert.equal(filter.wantFile('file'), true);
```

Comment:
it is slightly confusing to me that this is `true`. I guess it's because `file` is assumed to be relative to `sourceDir` and not relative to `.`?